packages/@ember/-internals/glimmer/lib/templates/root.ts

```diff
@@ -1,4 +1,5 @@
 import { precompileTemplate } from '@ember/template-compilation';
 export default precompileTemplate(`{{component this}}`, {
   moduleName: 'packages/@ember/-internals/glimmer/lib/templates/root.hbs',
+  strictMode: true,
```

Comment:
Notice that this benefits from being strictMode and *does not* need anything from scope. Hence the type declaration change in this PR.